test/resources/invalid/parse/ImportBImportsC.flo

```diff
@@ -0,0 +1,3 @@
+import './ImportC' as ImportC;
+
+action UPDATE_USER = network(url = ImportC.UPDATE_USERS_URL, method = 'PUT');
```

Comment:
Note to @JeffreyDoyle: We need a syntax for specifying bodies of a POST/PUT/PATCH/DELETE network action.

Reply:
Maybe something like:
```
// Note how body accepts a js type
action UPDATE_USER = network(url = 'blah/blah/blah', method = 'PUT', body = `{
  field1: "My Field",
  field2: 123,
  field3: {
    field4: true,
    field5: [1, 2, 3]
  }
}`)